app/frontend/src/features/profile/ProfileTagInput.tsx

```diff
@@ -0,0 +1,229 @@
+import React, { useState, useEffect, useRef } from "react";
+import {
+  Checkbox,
+  IconButton,
+  InputBase,
+  ButtonBase,
+  Popper,
+  fade,
+  makeStyles,
+  createStyles,
+} from "@material-ui/core";
+import Autocomplete, {
+  AutocompleteCloseReason,
+} from "@material-ui/lab/Autocomplete";
+import { CloseIcon, ExpandMore } from "../../components/Icons";
+
+const useStyles = makeStyles((theme) =>
+  createStyles({
+    popper: {
+      borderRadius: theme.shape.borderRadius,
+      borderWidth: 1,
+      borderStyle: "solid",
+      borderColor: "rgba(0, 0, 0, 0.23)",
+      boxShadow: theme.shadows[3],
+      zIndex: 1,
+      backgroundColor: theme.palette.background.default,
+      marginTop: theme.spacing(1),
+    },
+    button: {
```

Comment:
the normal buttons have a hover effect, would be good to copy that

Reply:
Right! I chose to match the default input instead. Had to resort to boxShadow instead of border to prevent shifting. Please have a look!